MiniApp/Classes/Utilities/UrlParser.swift

```diff
@@ -3,20 +3,28 @@
  */
 struct UrlParser {
 
+    static let separatorKey: String = "map-published/"
+    static let dropLimit = 2
+
     /**
      * Parses for the file directory from an URL provided in a manifest.
      * @param { url: String } - URL to parse through.
      * @return { String } - The file directory after removing the base URL.
      */
-    static func parseForFileDirectory(with url: String) -> String? {
-        guard let versionId =
-            url.components(separatedBy: "version/").last?.components(separatedBy: "/").first,
-            let directory = url.components(separatedBy: "\(versionId)/").last
-        else {
-            MiniAppLogger.e("MiniAppSDK: Failed parsing URL.")
-            return nil
+    static func getFileStoragePath(from url: String) -> String? {
+        if url.contains(separatorKey) {
```

Comment:
`contains` & `components` might lead to the same logical outcome (making the lookup by `contains` redundant). What do you think?

Reply:
I was thinking the same as you @ts-nitish-a-jain at first, but if the string does not contain the separator, the pathComponents will still have one element. So it's not the same logic. Maybe pathComponents size could be tested.

Reply:
Yes, @Climbatize is right. Even if components couldn't find an element (provided with separator) it will still return an element.